python/ray/rllib/policy_gradient/rollout.py

```diff
@@ -5,8 +5,8 @@
 import numpy as np
 import ray
 
-from reinforce.filter import NoFilter
-from reinforce.utils import flatten, concatenate
+from .filter import NoFilter
```

Comment:
I think TensorFlow would do `from ray.rllib.policy_gradient.filter import NoFilter`. That may be a bit cleaner, what do you think?